### ISSUE TITLE
v. 1.069 Italic caron with acute and breve with acute

### ISSUE THREAD

**Issue by ischaap** [psb1558/Junicode-font#211]
Glyphs with caron and acute look good in roman, but the italic acute looks it could use adjustment:
![image](https://github.com/psb1558/Junicode-font/assets/139657890/642e11cd-fb97-4b56-b761-6697b0ce7c80)

Breve with acute also looks good in roman, but in italic, the acute looks a little too high:
![image](https://github.com/psb1558/Junicode-font/assets/139657890/8c80421a-52d0-44e0-b59a-62178c9a54ee)

**Comment by psb1558** [comment]
I've been working on regularizing the positions of anchors on combining marks, so that the upper marks in your example always have the same position:
<img width="72" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/49c290eb-ed71-49e3-ab21-a70fc6e00477">
For some combinations of marks Junicode uses compound glyphs—especially for the Vietnamese stacked diacritics. But these compound glyphs aren't just for Vietnamese: a compound, for example, of breve and acute is substituted for the sequence U+0306 (combining breve) and U+0301 (combining acute):
<img width="63" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/ecbb691d-57c8-42c0-817f-c77f110dd7bc">
Where this doesn't work it's because of normalization, which happens before OpenType features get applied. This can be defeated with U+034F (combining grapheme joiner):
<img width="78" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/95490c14-ad27-444d-81d8-20a070833024">
This will all be in Junicode 2.000beta1.

**Comment by ischaap** [comment]
Following up on this issue for v. 2.000beta3 static ttf in Word for 365.

The caron and acute look consistent now, and so do the breve and acute with _s_, as in your example. But I just cant get them to work with any vowel other than _a_, even with the combining grapheme joiner. I've shown the sequence of my input below -- and I tried two variations in the sequencing of characters in case I'm just doing it wrong (??).

In both case the second e is italicized. The comma in the second example shows up after italicizing. Prior to italicizing, it looks just like the one to its left (box with approximately correct-looking breve and acute).

![image](https://github.com/psb1558/Junicode-font/assets/139657890/4919e1ff-8d93-4568-897c-ff811ff3b190)

**Comment by psb1558** [comment]
It's working for me:
<img width="333" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/9c2c90cb-fe00-4301-8b26-bbeff59b258a">
But it won't work with precomposed characters: you use U+034F to keep the layout engine from substituting those precomposed characters. Also, it won't work with caron, but rather these combos:
- breve + grave
- breve + acute
- breve + hook
- circumflex + grave
- circumflex + acute
- circumflex + hook
- circumflex + tilde

There are lots of possible combos that could receive this treatment, but I have done this only for the Vietnamese combos. If you need stacked diacritics and they all need to harmonized, then prevent the formation of the Vietnamese combos, again by the judicious placement of U+034F:
<img width="362" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/49570fad-6bc9-40d7-b13b-7a37a79297f7">

Here only the **ắ** poses a problem, since **a** is the only Vietnamese letter that can take the breve + acute combo: I placed the U+034F between the breve and the acute: `a + U+0306 + U+034F + U+0301`.

**Comment by psb1558** [comment]
Oh, and I don't know what to say about a stray comma turning up. A font can't do that by itself, but only some bizarre interaction between font and software.

**Comment by ischaap** [comment]
Ahh, I see the problem -- it's my total ignorance of Vietnamese language and typography. I didn't realize that in Vietnamese the acute is positioned a bit lower relative to the breve -- I was expecting the higher position in all contexts. When I opened this issue I didn't have Vietnamese in mind at all, only Proto-Slavic, Latin, and ad hoc uses: in my example of _tĕ́nĕt_, I was trying to imply a transcription of the Latin word meant to show that it has two short vowels and stress on the first syllable. Then I also failed to notice that in a build subsequent to my first post you did normalize the non-Vietnamese (higher) position of the acute relative to the breve in roman and italic.

Also, thanks for the code showing exactly where U+034F fits in the sequencing -- that was helpful. This is what it produces in Word for 365 (but weirdly only if you use the Insert Symbol method -- you can't just type the codes and press Alt+x, or you just get a box):
![image](https://github.com/psb1558/Junicode-font/assets/139657890/2dfb44ef-b939-4f10-bd22-7358b8ee8ffc)

At this point I felt like I was going a bit crazy so I resubbed to InDesign and of course it works perfectly there:
![image](https://github.com/psb1558/Junicode-font/assets/139657890/5d2d8856-af42-44c2-ae96-136001f9bae7)

So long story short, I think this was a combo of my ignorance and Word for 365's bugginess -- apologies @psb1558 for the confusion!

**Comment by psb1558** [comment]
I think this is done now.
